src/lib/download.ts

```diff
@@ -22,6 +30,9 @@ export function downloadBlob(blob: Blob, fileName: string) {
   anchor.click()
   window.URL.revokeObjectURL(anchor.href)
   document.body.removeChild(anchor)
+  if (callback) {
```

Comment:
This is unnecessary. There are no deferred statements.

Reply:
True, fixed!